ansible/roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# defaults file for role/docker
+
+install_docker: "{{ install_docker }}"
+docker_edition: "ce"
+docker_pkg: "docker-{{ docker_edition }}"
+
+docker_apt_ignore_key_error: True
+docker_apt_key: "9DC858229FC7DD38854AE2D88D81803C0EBFCD88"
```

Comment:
what is this key?

Reply:
It's a docker apt-key, it's on docker's documentation